src/dag/README.md

```diff
@@ -148,3 +148,87 @@ forEachTopologicalGeneration(graph, generation => {
   console.log(generation);
 });
 ```
+
+### forEachTransitiveRelation
```

Comment:
Maybe explain that the function will not test whether the given graph is actually a DAG?